src/ui/pages/Security/ChangePassword.tsx

```diff
@@ -0,0 +1,173 @@
+  const navigate = useNavigate();
+  const { data: user } = useQuery({
+    queryKey: ['account/getExistingUser'],
+    queryFn: () => accountPublicRPCPort.request('getExistingUser'),
+  });
+  const changePasswordMutation = useMutation({
+    mutationFn: async ({
+      currentPassword,
+      newPassword,
+    }: {
+      currentPassword: string;
+      newPassword: string;
+    }) => {
+      invariant(user, 'User must be defined');
+      await accountPublicRPCPort.request('changePassword', {
+        user,
+        currentPassword,
+        newPassword,
+      });
+    },
+    onSuccess: async () => {
+      zeroizeAfterSubmission();
+      navigate('./success');
+    },
+  });
+  const [formData, setFormData] = useState<FormData | null>(null);
+  const submitError = changePasswordMutation.isError
+    ? getError(changePasswordMutation.error)
+    : null;
+  const incorrectPassword = submitError?.message === 'Incorrect password';
+  return (
+    <PageColumn>
+      <PageTop />
+      <PageHeading>Change Password</PageHeading>
+      <PageTop />
+      <form
+        onChange={(event) => {
+          const fd = new FormData(event.currentTarget);
+          setFormData(fd);
+        }}
+        onSubmit={(event) => {
+          event.preventDefault();
+          const fd = new FormData(event.currentTarget);
+          const currentPassword = fd.get('currentPassword') as
+            | string
+            | undefined;
+          const newPassword = fd.get('newPassword') as string | undefined;
+          invariant(currentPassword, 'currentPassword is required');
+          invariant(newPassword, 'newPassword is required');
+          changePasswordMutation.mutate({ currentPassword, newPassword });
+        }}
+      >
+        <VStack gap={24}>
+          <label style={{ all: 'unset' }}>
+            <VStack gap={4}>
+              <UIText kind="body/regular">Current password</UIText>
+              <VStack gap={4}>
+                <Input
+                  autoFocus={true}
+                  type="password"
+                  name="currentPassword"
+                  placeholder="Password"
+                  required={true}
+                />
+                {incorrectPassword ? (
+                  <UIText kind="caption/regular" color="var(--negative-500)">
+                    Incorrect Password
+                  </UIText>
+                ) : null}
+              </VStack>
+            </VStack>
+          </label>
+          <label style={{ all: 'unset' }}>
+            <VStack gap={4}>
+              <UIText kind="body/regular">Enter new password</UIText>
+              <VStack gap={4}>
+                <Input
+                  type="password"
+                  name="newPassword"
+                  placeholder="Set new or verify existing one"
+                  required={true}
+                />
+              </VStack>
+            </VStack>
+          </label>
+          <label style={{ all: 'unset' }}>
```

Comment:
Can we refactor all `Unstyled***` components with this?
Not for this task, just interesting :)